src/web/app/pages-instructor/instructor-course-student-details-page/instructor-course-student-details-page.component.ts

```diff
@@ -21,18 +22,23 @@ interface StudentDetails {
   styleUrls: ['./instructor-course-student-details-page.component.scss'],
 })
 export class InstructorCourseStudentDetailsPageComponent implements OnInit {
-
```

Comment:
Unrelated change here, most files do leave one line spacing before the declaration of variables so let's do that here as well :)

Reply:
Done